### ISSUE TITLE
Unexpected case: 7, DP4

### ISSUE THREAD

**Issue by prashantpandey** [samtools/bcftools#1085]
I am trying to merge some vcf files (protected, so can't share) and get this error.
Could you please tell what this error means? If it's something trivial that can I can fix in the VCF file then I can do that and proceed.

Thanks,
Prashant

**Comment by pd3** [comment]
This cryptic error is thrown when non-numeric FORMAT fields are merged and their `Number=` definitions in the header are not supported yet. Cannot really say for sure without seeing a reproducible test data.

**Comment by rsteinfe1** [comment]
Hi, 

I get a similar error when I try to merge multiple VCF files into one. I was wondering if we could open this ticket again?
I created a minimalistic reproducible example.

I tried to run:
`
bcftools --version
bcftools 1.11
Using htslib 1.11
Copyright (C) 2020 Genome Research Ltd.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

tabix -p vcf Subset1.vcf.gz 
tabix -p vcf Subset2.vcf.gz
bcftools merge -Oz -o test.vcf.gz Subset1.vcf.gz Subset2.vcf.gz
[W::bcf_hdr_merge] Trying to combine "HDS" tag definitions of different types
Unexpected case: 7, HDS
` 
I can get rid of the warning, by changing the vcf meta line, but it's not causing the Unexpected case error.

Files  (I can't upload tabix index files to GitHub):
[Subset2.vcf.gz](https://github.com/samtools/bcftools/files/6422907/Subset2.vcf.gz)
[Subset1.vcf.gz](https://github.com/samtools/bcftools/files/6422908/Subset1.vcf.gz)
